doc/user_guide/API.rst

```diff
@@ -550,7 +510,6 @@ Low-Level Schema Wrappers
    ValueDefWithConditionMarkPropFieldOrDatumDefstringnull
    ValueDefWithConditionStringFieldDefText
    ValueDefnumber
-   ValueDefnumberExprRef
    ValueDefnumberwidthheightExprRef
    Vector2DateTime
    Vector2Vector2number
```

Comment:
It looks like some of the new classes are missing from the docs, in particular new encoding channels like `XOffset`. We'll also want to add these to the encodings tables elsewhere in the documentation, and perhaps add an example or two (maybe a jitter chart and/or a grouped bar chart?)

Reply:
We could probably do that in a followup PR if you prefer

Reply:
Thanks for the feedback!  I will try to get these added and will let you know if I get stuck.

Should I systematically go through the old examples and make sure they aren't using deprecated terminology?  I know that eventually needs to be done, but if you tell me it's smarter to wait, that also makes sense.

Reply:
I think it's fine to leave those old examples for now, one benefit is that they act as tests that deprecated APIs still work correctly!

Reply:
I tried to incorporate those changes (include missing classes, examples for XOffset) in the most recent few commits.  Please let me know if you notice anything else, or if any of these seem incorrect!